docs/How/using_project_mu.md

```diff
@@ -0,0 +1,108 @@
+# How to setup a new Repo for a Platform that will use Project MU?
+
+This document will describe the base guidelines for setting up a Project MU repo.
+
+You will need:
+
+1) Git
+2) Python 3.7
+3) A text editor
+4) Look at [layout](../WhatAndWhy/layout.html) to understand our recommended repository layout. You can also look at [ms-iot iMX8](https://github.com/ms-iot/MU_PLATFORM_NXP) for a real platform implementation.
+
+## 0) Nomenclature
+
+I will use the term *workspace root* to reference the base folder for your code tree.
+Ordinarily, we use the *Platform Repository* as the outer-most layer. This means that the outermost git repository is where we store Platform specific files and libraries. In this case, our *Platform Repo* is also our *workspace root*. If you choose to have a different repository layout, it will be important to note what your *workspace root* is, as it should still be the base folder of your code tree.
+
+## 1) Create Git Repo
+
+Make new directory.
+```cmd
+mkdir NewPlatformRepo
+cd NewPlatformRepo
+git init
+```
+
+This will serve as our Platform Repository as well as our Workspace Root.
+
+For more information on creating a Git repo, here are [command line instructions](https://kbroman.org/github_tutorial/pages/init.html) and here are [web instructions](https://help.github.com/en/articles/create-a-repo).
+
+## 2) Add pertinent submodules
+
+Project MU is separated into submodules. For each submodule that you need for your project, run the "git submodule add" command to add it to your base Repository. The path after the URL is the path we typically use to group the submodules. You can change it if you'd like, just remember your environment will diverge from the one in these instructions.
+
+### [MU_BASECORE](https://github.com/Microsoft/mu_basecore.git)
+
+This is the main repo. Contains the guts of the build system as well as core UEFI code, forked from TianoCore. You will need this to continue.
```

Comment:
Maybe core repo would be better. Main makes me think this will be the base of my project